NMS-15218: Update path outage docs (#5661)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/service-assurance/path-outages.adoc

[[ga-service-assurance-path-outage]]
= Use Path Outages to Suppress Notifications

An outage of a core network component can cause a lot of node outages.
You can use path outages to suppress notifications based on how nodes depend on each other in the network, as defined by a critical path.
The critical path needs to be configured from the network perspective of the monitoring system.

== Enable path outages

By default, the path outage feature is disabled and has to be enabled for pollerd to check if notifications should be suppresed for an outage.
Edit the `$\{OPENNMS_HOME}/etc/poller-configuration.xml` file to set `pathOutageEnabled` to `true` at the top of the file.

> A service restart is required to pick up changes to this setting.

Reviewer: ```suggestion
A service restart is required after changing this setting.
```

In this case, an administrator would like to receive just one notification for `default-gw-01`; notifications for the other nodes behind it should not be generated.
Building this configuration in {page-component-title} requires the following information:

Parent Foreign Source:: The foreign source where the parent node is defined.
If the parent node is in the same requisition as the child node, this field can be left blank.
Parent Foreign ID:: The foreign ID of the parent node this node depends on.

> It is possible to define the parent node based on the label, though using the ID is more reliable in the case the node label changes in the future.

Reviewer 2: ```suggestion
It is possible to define the parent node based on the label, though using the ID is more reliable in case the node label changes in the future.
```
